server/Arcadia.Assistant.Web/Startup.cs

```diff
@@ -149,13 +158,24 @@ public void ConfigureContainer(ContainerBuilder builder)
 
         // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
         // ReSharper disable once UnusedMember.Global
-        public void Configure(IApplicationBuilder app, IHostingEnvironment env, ISecuritySettings securitySettings)
+        public void Configure(
+            IApplicationBuilder app,
+            IHostingEnvironment env,
+            ISecuritySettings securitySettings,
+            ILifetimeScope container)
         {
             if (env.IsDevelopment())
             {
                 app.UseDeveloperExceptionPage();
             }
 
+            // ReSharper disable once ObjectCreationAsStatement
+            new AutoFacDependencyResolver(container, this.ActorSystem);
+
+            this.ActorSystem.ActorOf(
```

Comment:
why have ActorSystem as a field when you can inject it here?

Reply:
I couldn't recall what I missed. That's it :)

Reply:
Reverted.